proto/stride/interchainquery/v1/genesis.proto

```diff
@@ -7,6 +7,12 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/Stride-Labs/stride/v9/x/interchainquery/types";
 
+enum TimeoutPolicy {
+  REJECT_QUERY_RESPONSE = 0;
```

Comment:
nit: do we need this enum entry if it's not used? maybe there's a future use case for it

Reply:
I was originally using it and then changed it last minute haha. This is our current behavior on mainnet. Once the three of us land on timeout policies for each callback, if this is not used we can remove it